Move "Report an Issue" item to sidebar-menu.

dlaliberte opened the pull request:
This PR fixes the keyboard navigation issue reported in #294 with the following changes:

* Move the second sidebar sl-tree into the sidebar-menu component
* Comment out the display:none statistics item since it also interferes with navigation.
* Remove the divider, since it does not appear to be allowed in the sl-tree.

dlaliberte commented:
I could add a keyboard navigation test via playwright.

jcscottiii reviewed (changes requested):
You should use sl-divider

![image](https://github.com/GoogleChrome/webstatus.dev/assets/7788930/f42c8c39-2bb1-495c-8138-3b03df4272e9)


Also the styling on the `Report an issue` is showing the blue href style now. That should be changed to keep the text the same style.

jrobbins reviewed (approved):
This change looks OK to me.  And, it seems to fix the reported problem.

I did like have some separation between the main navigational items vs. the customer service type of items.   Could we have a some extra padding or something to just visually separate them?  (My first thought was to have a blank tree item, but we don't want to add a useless tab stop.)

dlaliberte commented:
I had removed the divider since it appeared to be interfering with keyboard navigation, but I added it back in now, and it does seem to work, so I'm not sure what else might have changed.

I also moved/fixed the styling to be pretty much as before.

dlaliberte commented:
> You should use sl-divider
> 
> ![image](https://private-user-images.githubusercontent.com/7788930/335348844-f42c8c39-2bb1-495c-8138-3b03df4272e9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcxMDMwMzYsIm5iZiI6MTcxNzEwMjczNiwicGF0aCI6Ii83Nzg4OTMwLzMzNTM0ODg0NC1mNDJjOGMzOS0yYmIxLTQ5NWMtODEzOC0zYjAzZGY0MjcyZTkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MzBUMjA1ODU2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWNkNjgyODM0ODc3ODdjNzdiMjdjNWM5YzJjNTNjZTJkMjI5Y2Q4NjU5ZjBkODBlYTU3MDc1M2VlNDkwOTdiZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.37I1UlTmlRM476lvFmacikoqJE44hQczqqm0V8Glikw)
> 

With the sl-divider inside the tree, it appears right, but I get this error:
`Element has children which are not allowed: [role=separator]`  But fortunately, it looks like I can add aria-hidden="true" to the sl-divider and this makes it ok.
